modules/tile-converter/src/deps-installer/deps-installer.ts

```diff
@@ -1,7 +1,7 @@
 import {load, fetchFile} from '@loaders.gl/core';
 import {ZipLoader} from '@loaders.gl/zip';
 import {writeFile} from '../lib/utils/file-utils';
-import {join} from 'path';
+import {join, dirname} from 'path';
```

Comment:
Nit: I think loader-utils may have implementations of these that also work on browser, but this is of course fine for node.